content/en/docs/tasks/traffic-management/egress/egress-gateway/test.sh

```diff
@@ -58,11 +58,31 @@ _verify_contains snip_egress_gateway_for_http_traffic_7 "HTTP/2 200"
 
 # Verify routing through gateway
 if [ "$GATEWAY_API" == "true" ]; then
-    _verify_contains snip_egress_gateway_for_http_traffic_10 "outbound|80||edition.cnn.com"
+    _verify_contains snip_egress_gateway_for_http_traffic_11 "outbound|80||edition.cnn.com"
 else
     _verify_contains snip_egress_gateway_for_http_traffic_8 "outbound|80||edition.cnn.com"
 fi
 
+# Verify the certificate is correct if mTLS is enabled
+set +e  # Don't exit on failure
+kubectl apply -f - <<EOF
+apiVersion: security.istio.io/v1beta1
+kind: PeerAuthentication
+metadata:
+  name: "default"
+  namespace: "istio-system"
+spec:
+  mtls:
+    mode: STRICT
+EOF
+set -e  # Exit on failure
+if [ "$GATEWAY_API" == "true" ]; then
+    _verify_contains snip_egress_gateway_for_http_traffic_13 "$snip_egress_gateway_for_http_traffic_13_out"
+else
+    _verify_contains snip_egress_gateway_for_http_traffic_10 "$snip_egress_gateway_for_http_traffic_10_out"
```

Comment:
Hmm, looks like a bug in the framework. Notice that the `...10_out` snip that you're using for the expected output parameter does have the first line of output indented:
```
! read -r -d '' snip_egress_gateway_for_http_traffic_10_out <<\ENDSNIP
            X509v3 Subject Alternative Name: critical
                URI:spiffe://cluster.local/ns/istio-system/sa/istio-egressgateway-service-account
ENDSNIP
```
but the error message from `_verify_same` seems to have lost the indent.

I guess you have 2 options here:

1. Dig into the framework and fix this "expected output indentation" bug.
2. Switch back to using `verify_contains`, since that works in this case because the indentation of the second line of output is correct.

If you have time to do it, I'd prefer option 1 😃

Reply:
I'll open a separate GH issue for option 1 and revert back to `verify_contains`. Once I discover the issue in the framework I will update this file as part of it.

Reply:
Great, thanks!